run.sh

```diff
@@ -256,8 +238,55 @@ function run_cloud_build(){
     echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Cloud Build Initialization is finished."
 }
 
+# Init script with proper message and display active user account
+
+function gcloud_init_confs(){
+    user_active_account="$(gcloud auth list --filter=status:ACTIVE --format="value(account)")"
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Your active account [${user_active_account}]"
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Choose Project ID"
+    _choose_and_set_project_id
+	echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Project ID was updated."
+}
+
+
+# Choose and set project id
+# Error:
+#   Exit Code 1
+function _choose_and_set_project_id(){
+    array_projects=()
+    project_id=""
+    count=0
+    for project in  $(gcloud projects list --format="value(projectId)")
+    do
+        count=$((count + 1))
+        echo "[$count]:  $project"
+        array_projects+=("$project")
+    done
+    read -n 1 -p "Please fill in number of project: " mainmenuinput
```

Comment:
Please fill in number of project where you would like the integration to be deployed in